src/ex1/style.css

```diff
@@ -0,0 +1,141 @@
+body {
+    background: linear-gradient(skyblue,rgb(79, 79, 255));
+    background-attachment: fixed;
+}
+* {
+    box-sizing: border-box;
+}
+.container {
+    width: 30%;
+    background-color:white;
+    margin: auto;
+    padding: 16px;
+    box-shadow: 20px 20px 20px 0px rgba(0, 0,0, 0.3);
+    font-family: cursive;
+    display: grid;
+    border-radius: 10px;
+}
+.title {
+    margin-top: auto;
+    margin-bottom: 5px;
+}
+.subTitle {
+    opacity: 0.7;
+}
+
+input {
+    padding: 12px;
+    width: 80%;
+    font-family: cursive;
+}
+.addBtn {
+    border-radius: 4px;
```

Comment:
most of the buttons share the same CSS properties- it's better to have a standard button class (same color, bg color, border radius etc) and add another class for specific use cases